### PR TITLE
Add kube-aws, change base image, update paths

### PR DESCRIPTION
**Pull request by SweetOps**
Changes:

* Add kube-aws 
* Base image has been changed to `alpine`
* Update paths

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,54 @@
...
+ENV JINJA2_VERSION 2.10
+ENV AWSCLI_VERSION 1.11.185
+ENV KUBE_AWS_RELEASE v0.9.8
+ENV KUBECTL_VERSION v1.8.5
+
+RUN apk update \
+    && apk add --no-cache unzip curl tar bash \
+    python make bash vim jq  \
+    openssl openssh-client sshpass  \
+    gcc libffi-dev python-dev musl-dev openssl-dev py-pip py-virtualenv \
```
Comment (SweetOps) [inline Dockerfile:12]: `gcc` need only for packages building, so it can be removed below

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir /mnt/opsbox/.aws /mnt/opsbox/.kube
```
Comment (SweetOps) [inline start.sh:3]: you've created symlinks in Dockerfile , so for what it needs? :)

File: Dockerfile (hunk 1 above)
Comment (const-bon) [reply Dockerfile:12]: Python packages installation reqires `gcc`

File: start.sh (hunk 2 above)
Comment (const-bon) [reply start.sh:3]: Please read README.md.
It is required for storing all credetials used in container as a separate folder n the host.

File: Dockerfile (hunk 1 above)
Comment (SweetOps) [reply Dockerfile:12]: it can be removed after installation of packages:
```
RUN apk add gcc 
RUN pip something 
RUN apk del gcc
```

Comment (SweetOps) [reply Dockerfile:12]: nvm difference in ~5mb

Comment (it-svit) [reply Dockerfile:12]: It won't bring much benefits.

**Review by it-svit — APPROVED** [review]

**Pull request merged**